pkg/datasource/csvsource/types.go

```diff
@@ -0,0 +1,87 @@
+package csvsource
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+)
+
+type CsvTick struct {
+	Timestamp       int64            `json:"timestamp"`
+	Symbol          string           `json:"symbol"`
+	Side            string           `json:"side"`
+	TickDirection   string           `json:"tickDirection"`
+	Size            fixedpoint.Value `json:"size"`
+	Price           fixedpoint.Value `json:"price"`
+	HomeNotional    fixedpoint.Value `json:"homeNotional"`
+	ForeignNotional fixedpoint.Value `json:"foreignNotional"`
+}
+
+type SupportedExchange int
+
+const (
+	Bybit   SupportedExchange = 0
+	Binance SupportedExchange = 1
```

Comment:
you could use the defined exchange names in the pkg/types package

Reply:
thx for pointing out